### DISCUSSION TITLE
RaspAP on Ubuntu 24.04 LTS

### DISCUSSION THREAD

**Discussion by MariusQuabeck** [RaspAP/raspap-webgui#1637] [Q&A] [answered]
Hi 🖖 

Does anyone know if support for Ubuntu 24.04 LTS is in the works?
I have a project where I'd like to use RaspAP and I'd really like to build it on 24.04, ideally without docker.

I know better than to ask for timelines, just wondering if this is something I should wait for or start exploring other options.

Thanks :)

**Comment by billz** [comment, marked as answer]
Our [project roadmap](https://github.com/orgs/RaspAP/projects/2) is open and public. We currently have a backlog of new features. Ubuntu 23.04 (Lunar) support is in beta and no testing has been done on 24.04. You're welcome to contribute to the effort to extend to support for this distro. This can be done by testing an install and opening an issue with your findings.
